### PR TITLE
feat: add more functionality to attention component

### PR DESCRIPTION
**Pull request by felicia-haggqvist**
**Part of fixes for Jira-tickets:** [WARP-432](https://nmp-jira.atlassian.net/browse/WARP-432), [WARP-341](https://nmp-jira.atlassian.net/browse/WARP-341) & [WARP-373](https://nmp-jira.atlassian.net/browse/WARP-373)

- Add `crossAxis` prop
- Only render attention component when `targetEl` is not undefined or when it is a `callout`
- Bump to latest version of `@warp-ds/core` dependency
- Add additional tests to `AttentionTest.tsx` to test for when `targetEl`is undefined or when `callout` is either true or false.

### CODE REVIEW

File: packages/attention/src/component.tsx [hunk 1]
```diff
@@ -137,54 +141,58 @@ export function Attention(props: AttentionProps) {
...
-            onClick={onDismiss}
-            onKeyDown={(event) => {
-              if (!props.onDismiss) return
-              if (event.key === 'Escape') {
-                props.onDismiss()
-              }
-            }}
-            className={ccAttention.closeBtn}
+    <>
+      {(props.callout || (props.targetEl !== undefined && !props.callout)) && (
```
Comment (BalbinaK) [inline packages/attention/src/component.tsx:145]: What do you think about doing the opposite here, too, and returning `null` if `!props.callout && props.targetEl === undefined`?

Comment (felicia-haggqvist) [reply packages/attention/src/component.tsx:145]: Yes, I was thinking of doing the same here as well! 🎉 However, not sure if I should make the same changes for the vue attention component: https://github.com/warp-ds/vue/pull/161/files#diff-73fbe20fc6b40d2120b1a5e414f088ed51cea513e1eb9745402ee02a639b22b0R220

Comment (BalbinaK) [reply packages/attention/src/component.tsx:145]: Yeah, sounds like Vue should stay as is with the v-if syntax.

Comment (felicia-haggqvist) [reply packages/attention/src/component.tsx]: Updated in this commit: [5c799d2](https://github.com/warp-ds/react/pull/233/commits/5c799d27f0224b55b99cfc94b1f81b924506a6d8)

File: packages/attention/src/component.tsx [hunk 2]
```diff
@@ -137,54 +141,58 @@ export function Attention(props: AttentionProps) {
...
-            )}
-            onClick={onDismiss}
-            onKeyDown={(event) => {
-              if (!props.onDismiss) return
-              if (event.key === 'Escape') {
-                props.onDismiss()
-              }
-            }}
-            className={ccAttention.closeBtn}
+    <>
```
Comment (BalbinaK) [inline packages/attention/src/component.tsx:144]: I don't think we need this? It would at least make the diff much smaller :D

Comment (felicia-haggqvist) [reply packages/attention/src/component.tsx:144]: I tried to remove `<><>` but I get this error then: `This expression is not callable. Type '{}' has no call signatures.ts. Property assignment expected.` 😕

Comment (BalbinaK) [reply packages/attention/src/component.tsx:144]: The curly braces (`{}`) can be removed from the return statement when you remove `<>`, and that should fix it :)

Comment (felicia-haggqvist) [reply packages/attention/src/component.tsx]: Updated in this commit: [7039695](https://github.com/warp-ds/react/pull/233/commits/7039695d139f54d1f290b4157e5ee0842a166128)

**Review by BalbinaK — APPROVED** [review]

**Pull request merged**